### DISCUSSION TITLE
Crowdin translation contribution

### DISCUSSION THREAD

**Discussion by elvissaba** [YetiForceCompany/YetiForceCRM#14360] [Wishlist]
Can you list Crowdin translators as contributors on GitHub project and on your webpage?

**Comment by mariuszkrzaczkowski** [comment]
good idea, do you have any ideas how and where to do it?

**Reply by elvissaba** [reply]
Pestashop community is good example:
https://contributors.prestashop.com/
https://translators.prestashop.com/
https://build.prestashop.com/news/do-you-speak-prestashop-february-2021/

**Reply by mariuszkrzaczkowski** [reply]
we will check

crowdin Top Members report
![image](https://user-images.githubusercontent.com/10198654/110531859-634d2680-811c-11eb-8a1f-bc03f3e932e5.png)
